01-node-tutorial/.gitignore

```diff
@@ -1,4 +1,5 @@
 /node_modules
```

Comment:
I see node modules in this commit - did you add the gitignore afterwards? No worries if so, might want to try following the instructions here to remove committed files that have been got ignored after the fact :) 

https://stackoverflow.com/questions/7927230/remove-directory-from-remote-repository-after-adding-them-to-gitignore

Reply:
I think I've fixed this. Please let me know if I've got it addressed or if not if we can set up a meeting to address.
Thanks so much!